src/secret_base.c

```diff
@@ -852,7 +852,7 @@ void sub_80E9C2C(void)
 
 void sub_80E9C74(void)
 {
-    CreateTask(sub_8126AD8, 0);
+    CreateTask(SecretBasePC_InitSecretBaseDecorateMenuTask, 0);
```

Comment:
Task names should start with the `Task_` prefix, rather than have a `...Task` postfix.